index.js

```diff
@@ -46,6 +46,9 @@ module.exports = fp(async function (app, { all, policy, ttl, cacheSize, skip, st
       cacheSize
     })
   }
+}, {
+  fastify: '>=3.x',
```

Comment:
```suggestion
  fastify: '^3.x',
```

Reply:
in auth it's been done with the >= specifier. any reason why we should do it one way or the other?

https://github.com/mercurius-js/auth/blob/main/index.js#L32

Reply:
Using the caret is better as it will not support Fastify v4 when it come out without manual change.